src/_includes/partials/components/filter-resources.njk

```diff
@@ -11,7 +11,7 @@
             {% for topic in resourceTopics.resourceTopics %}
                 <li>
                     <input class="filter-checkbox" type="checkbox" data-filter="topics" id="{{ topic.value }}" name="{{ 'to_' + topic.value }}">
-                    <label for="{{ topic.value }}">{{ topic.label }}</label>
+                    <label aria-label="Topics filter, {{ topic.label }}" for="{{ topic.value }}">{{ topic.label }}</label>
```

Comment:
I'd suggest removing these, and instead enclosing each `<ul>` in a `<fieldset>` with a visually hidden `<legend>`:

```njk
<fieldset>
        <legend class="visually-hidden">{{ filterTitle }}</legend>
	<!-- <ul> goes here -->
</fieldset>
```